### DISCUSSION TITLE
Website: ISO Date

### DISCUSSION THREAD

**Discussion by mantissa7** [zed-industries/zed#14967] [Feature Requests]
The Releases and Blog pages both display dates in American format m/d/y which is difficult to differentiate from the d/m/y used by most of the world. 
https://en.wikipedia.org/wiki/List_of_date_formats_by_country

Since it is difficult to differentiate it would best to display dates using the ISO 8601 standard, as even if `Date.toLocaleDate()` is used it still takes time to mentally parse whether the date is MDY or DMY.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
